rom/dev/src/flow/cold_reset/fw_processor.rs

```diff
@@ -19,7 +19,6 @@ use crate::key_ladder;
 use crate::pcr;
 use crate::rom_env::RomEnv;
 use crate::run_fips_tests;
-#[cfg(not(feature = "no-cfi"))]
```

Comment:
Shouldn't these features stay (here and all below files)? The no-cfi feature should still be supported

Reply:
I can add them back.